src-cpp/rdkafkacpp_int.h

```diff
@@ -498,11 +554,22 @@ class ConfImpl : public Conf {
       return Conf::CONF_OK;
     }
 
+  Conf::ConfResult get(CertVerifyCb *&ssl_cert_verify_cb) const {
+      if (!rk_conf_)
+      return Conf::CONF_INVALID;
+      ssl_cert_verify_cb = this->ssl_cert_verify_cb_;
+      return Conf::CONF_OK;
+  }
 
+  Conf::ConfResult get(CertRetrieveCb *&ssl_cert_retrieve_cb) const {
+      if (!rk_conf_)
+      return Conf::CONF_INVALID;
```

Comment:
fix indent

Reply:
Same as above should we make this one different.